bin/garden-init

```diff
@@ -156,11 +165,29 @@ if [[ -v PKG_DIR ]]; then
 fi
 
 if [ -z "$nonDebian" ]; then
-	"$thisDir/garden-debian-sources-list" --gardenlinux-only \
-		$([ -z "$debianPorts" ] || echo '--ports') \
-		$([[ -v PKG_DIR ]] && echo --local-pkgs /run/packages) \
-		"$targetDir" "$suite" "$version"
-	cp "$keyringPlain" "$targetDir/etc/apt/trusted.gpg.d/gardenlinux.asc"
+
+	if [ $debianMirror -eq 1 ]; then
+
+		# Integrate Garden Linux repository with additional Debian native mirror
+		"$thisDir/garden-debian-sources-list" --gardenlinux-only --debian-mirror \
```

Comment:
Maybe we should refactor `garden-debian-source-list` so that:
- the `--gardenlinux-only` case is the default (not specifying --gardenlinux-only will result in the current `--gardenlinux-only` behavior
- remove `--gardenlinux-only` since it is the default
- if `--debian-mirror`  is specified then we can use the negation of `--gardenlinux-only` logic in `garden-debian-sources-list` which respects the `suite` parameter


**Motivation:**
specifying both; `--gardenlinux-only --debian-mirror` can be confusing


What do you think?

Reply:
I think this is a great idea and we should remove legacy code (e.g. `squeeze` etc.). I've just pushed some changes to prepare this for the outer scripts. If this is fine for you, I'd raise a new issue for refactoring the `garden-debian-sources-list` script (I've only adjusted the default for gardenlinux-repo usage).

Reply:
This makes sense for me. Thanks for creating the issue!